### PR TITLE
Add Turtle Script, Wyvern, Yoix, X10, Whiley, XOTcl, タイルズ, Snap!, Kojo, ThotPatrol, Rapira, PCASTL, Arturo and Stone and Rename Arnoldc to ArnoldC

### PR DESCRIPTION
**Pull request by gnuhead-chieb**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme
- [x] I have no association with the language
- [x] There are no copyright issues with this code
- [x] The language has not been added prior to this pull request

##### Turtle Script
https://apps.kde.org/kturtle/

##### Wyvern
https://wyvernlang.github.io/

##### Yoix
https://github.com/att/yoix

##### X10
http://x10-lang.org/

##### Whiley
http://whiley.org/

##### XOTcl
http://media.wu-wien.ac.at/index.html

##### タイルズ
https://vpltiles.com/

##### Snap!
https://snap.berkeley.edu/

##### Kojo
https://www.kogics.net/kojo

##### ThotPatrol
https://github.com/MindyGalveston/thotpatrol-

##### Rapira
https://freeduke33.github.io/rerap2/

##### PCASTL
http://www.pcosmos.ca/pcastl/

##### Arturo
https://arturo-lang.io/

##### Stone
https://github.com/chibash/stone

### CODE REVIEW

File: r/Rapira.rap [hunk 1]
```diff
@@ -0,0 +1 @@
+output: "Hello world";
```
Comment (MrBrain295) [inline r/Rapira.rap:1]:
```suggestion
output: "Hello World";
```

**Review by Richienb — APPROVED** [review]

File: x/X10.x10 [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+/**
+ * The classic hello world program, shows how to output to the console.
+ */
+public class HelloWorld {
+  public static def main(Rail[String]) {
+      Console.OUT.println("Hello World" );
+  }
+}
+
+
```
Comment (MrBrain295) [inline x/X10.x10:23]: Keep the copyright but remove the other comments.

**Review by MrBrain295 — CHANGES_REQUESTED** [review]

**Review by MrBrain295 — COMMENTED** [review]
It looks as though `ThotPatrol` is an interpreted language with a `.txt`file extension so I don't see why it is appearing as a binary executable in GitHub's file viewer.

**Review by MrBrain295 — COMMENTED** [review]
What is `.X10.x10.kate-swp`?

**Comment by gnuhead-chieb** [comment]
> It looks as though `ThotPatrol` is an interpreted language with a `.txt`file extension so I don't see why it is appearing as a binary executable in GitHub's file viewer.

ThotPatrol require UTF-16(Not UTF-8) for file encoding.otherwise we can not run the program.
so it cause github's bug.

> What is `.X10.x10.kate-swp`?

Ah,it mistaked.my text editor automaticlly created this file

**Review by MrBrain295 — APPROVED** [review]

**Pull request merged**